src/SortedVectors.jl

```diff
@@ -44,9 +42,12 @@ pop!(v::SortedVector) = pop!(v.data)
 popfirst!(v::SortedVector) = popfirst!(v.data)
 
 
-function resize!(v::SortedVector, n::Int)
-    resize!(v.data, n)
+function resize!(v::SortedVector, x)
```

Comment:
I'm not sure this should be called `resize!`.

Reply:
yes `resize!` can be a reasonable name because its deleting all of the values which are less than a particular number in array.

Reply:
Yes, but that is a very different meaning from what `resize!` usually means, where the argument is the new number of elements.